docs/reference/data-management/migrate-index-allocation-filters.asciidoc

```diff
@@ -0,0 +1,33 @@
+[role="xpack"]
+[[migrate-index-allocation-filters]]
+== Migrate index allocation filters to node roles
+
+If you currently use custom node attributes and routing allocation filters to move indices
+through data tiers in a hot-warm-cold architecture, we recommend that you switch to using node roles.
```

Comment:
I think we should link to the node filtering attributes and corresponding hot-warm-cold architecture blog post (https://www.elastic.co/blog/implementing-hot-warm-cold-in-elasticsearch-with-index-lifecycle-management) and the data tiers page.